editor/gui/editor_spin_slider.cpp

```diff
@@ -37,7 +37,7 @@
 #include "editor/editor_settings.h"
 
 String EditorSpinSlider::get_tooltip(const Point2 &p_pos) const {
-	if (grabber->is_visible()) {
+	if (grabber->is_visible() && !read_only) {
```

Comment:
```suggestion
	if (!read_only && grabber->is_visible()) {
```
Cheaper condition first here I'd suggest